docs/docusaurus/docs/deployment_patterns/how_to_instantiate_a_data_context_hosted_environments.md

```diff
@@ -34,7 +34,7 @@ The following guides will contain examples for each environment we have tested o
 Step 2: Create Expectation Suites and add Expectations
 -------------------------------------------------------
 
-If you want to create an Expectation Suite in your environment without using the CLI, you can follow this guide from step 5 onward to add a Datasource and an Expectation Suite: [How to connect to a PostgreSQL database](../guides/connecting_to_your_data/database/postgres.md#5-configure-your-datasource)
+If you want to create an Expectation Suite in your environment without using the CLI, you can follow this guide from step 5 onward to add a Datasource and an Expectation Suite: [How to connect to a PostgreSQL database](/docs/0.15.50/guides/connecting_to_your_data/database/postgres#5-configure-your-datasource)
```

Comment:
404 errors are returned when I attempt to use the updated link. Perhaps this is expected at this stage?

Reply:
The link worked for me when I clicked it inside the netlify preview -- can you reverify 404 on your end?